webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTable.js

```diff
@@ -258,7 +244,7 @@ class SubscriptionsTable extends Component {
 
     return (
       <LoadingState loading={subscriptions.loading} loadingText={__('Loading')}>
-        <ErrorAlerts
+        <SubscriptionsTableErrors
```

Comment:
Nice change here, but I wonder if we even need these alerts any longer because we have toast notifications for errors.  In fact, @tstrachota removed them in his PR, see https://github.com/Katello/katello/pull/7483#issuecomment-404235327.

Reply:
I think that we no longer need this inline alerts (but it would have been nice change indeed).

Reply:
:confused: 

Should we wait until one of our commit will get merge and the last one would rebase and remove it?

Reply:
@sharvit #7483 has been merged, you should be good to rebase now.